.gitignore

```diff
@@ -50,6 +50,7 @@ fastlane/
 iOSInjectionProject/
 #
 # Custom scripts
+fearless/env-vars.sh
```

Comment:
why do we need this?

Reply:
It contains Moonpay keys. I'll explain it later when Moonpay integration is ready